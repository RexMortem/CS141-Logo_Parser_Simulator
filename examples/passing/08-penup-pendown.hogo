forward 100
; We can raise the pen:
penup
forward 100
; And lower it again!
pendown
forward 100
