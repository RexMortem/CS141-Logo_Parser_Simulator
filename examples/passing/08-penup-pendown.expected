[GoForward 100, PenUp, GoForward 100, PenDown, GoForward 100]